src/handlers/changemakerFieldValuesHandlers.ts

```diff
@@ -0,0 +1,126 @@
+import { HTTP_STATUS } from '../constants';
+import {
+	db,
+	createChangemakerFieldValue,
+	loadBaseField,
+	loadChangemaker,
+	loadChangemakerFieldValueBatch,
+} from '../database';
+import {
+	isAuthContext,
+	isWritableChangemakerFieldValue,
+	Permission,
+	BaseFieldCategory,
+	BaseFieldSensitivityClassification,
+} from '../types';
+import {
+	FailedMiddlewareError,
+	InputValidationError,
+	InputConflictError,
+	NotFoundError,
+	UnprocessableEntityError,
+} from '../errors';
+import { authContextHasChangemakerPermission } from '../authorization';
+import { fieldValueIsValid } from '../fieldValidation';
+import type { Request, Response } from 'express';
+
+const postChangemakerFieldValue = async (
+	req: Request,
+	res: Response,
+): Promise<void> => {
+	if (!isAuthContext(req)) {
+		throw new FailedMiddlewareError('Unexpected lack of auth context.');
+	}
+
+	const body = req.body as unknown;
+	if (!isWritableChangemakerFieldValue(body)) {
+		throw new InputValidationError(
+			'Invalid request body.',
+			isWritableChangemakerFieldValue.errors ?? [],
+		);
+	}
+
+	const { changemakerId, baseFieldShortCode, batchId, value, goodAsOf } = body;
+
+	// Verify changemaker exists
+	await loadChangemaker(db, req, changemakerId).catch((error: unknown) => {
+		if (error instanceof NotFoundError) {
+			throw new InputConflictError('The changemaker does not exist.', {
+				entityType: 'Changemaker',
+				entityId: changemakerId,
+			});
+		}
+		throw error;
+	});
+
+	if (
+		!authContextHasChangemakerPermission(req, changemakerId, Permission.EDIT)
+	) {
+		throw new UnprocessableEntityError(
+			'You do not have write permissions on this changemaker.',
+		);
+	}
+
+	// Verify base field exists and validate its properties
+	const baseField = await loadBaseField(db, req, baseFieldShortCode).catch(
+		(error: unknown) => {
+			if (error instanceof NotFoundError) {
+				throw new InputConflictError('The base field does not exist.', {
+					entityType: 'BaseField',
+					entityShortCode: baseFieldShortCode,
+				});
+			}
+			throw error;
+		},
+	);
+
+	// Verify base field is organization category
+	if (baseField.category !== BaseFieldCategory.ORGANIZATION) {
+		throw new UnprocessableEntityError(
+			`Base field ${baseField.shortCode} has category '${baseField.category}' but must have category 'organization' for changemaker field values.`,
```

Comment:
This error should be thrown but the message should be changed.

Example when I tried to post:
 "message": "Base field proposal_primary_contact_title has category 'project' but must have category 'organization' for changemaker field values."

This message could be (mis)interpreted that something is wrong with the base field rather than something wrong with the post. Instead, clarify that `Values for ${baseField.shortCode} must be posted to /applicationForms as proposal field values because this field is of category '{baseField.category}'. Only 'organization' field values may be posted here.` (Or something like that)